hack/run-kiali.sh

```diff
@@ -280,6 +285,14 @@ else
   infomsg "You are connecting to a (non-OpenShift) Kubernetes cluster"
 fi
 
+# Port forward data for Istiod, used for the Istiod URL
+PORT_FORWARD_SERVICE_ISTIOD="service/istiod"
+LOCAL_REMOTE_PORTS_ISTIOD="15014:15014"
+if [ -z "${ISTIOD_URL:-}" ]; then
+  ISTIOD_URL= "${DEFAULT_ISTIO_URL}"
+fi
```

Comment:
You could replace this entire if-statement with a simpler single line:

```suggestion
ISTIOD_URL="${ISTIOD_URL:-${DEFAULT_ISTIOD_URL}}"
```

But definitely do change the variable name - needs a "D" in it.